guides/specification.md

```diff
@@ -513,6 +516,32 @@ Considering the previous scenario what happens is that:
 >     expect_same: facts.installed_rpm_version
 > ```
 
+### failure_message
+
+An optional failure message can be declare for every expectation.
+
+In case of an `expect` one, the failure message can interpolate `facts` and `values` present in the evaluation scope to provide more meaningful insights:
```

Comment:
I would put "of an `expect` expectation".
I know it looks redundant, but is reads better than with `one`.

Besides that, `evalution scope` looks a bit technical. What about? `can interpolate facts and values present in the check definitition`